azure_arc_servers_jumpstart/docs/gcp_terraform_windows.md

```diff
@@ -6,23 +6,39 @@ The following README will guide you on how to use the provided [Terraform](https
+
 # Create a new GCP Project
 
 * Browse to https://console.developers.google.com and login with your Google Cloud account. Once logged in, [create a new project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) named "Azure Arc Demo". After creating it, be sure to copy down the project id as it is usually different then the project name.
 
-* Once the new project is created and selected in the dropdown at the top of the page, you must enable Compute Engine API access for the project. Click on "+Enable APIs and Services" and search for "Compute Engine". 
+    ![](../img/gcp_windows/01.png)
 
-* Then click Enable to enable API access.
+    ![](../img/gcp_windows/02.png)
 
-* Next, set up a service account key, which Terraform will use to create and manage resources in your GCP project. Go to the [create service account key page](https://console.cloud.google.com/apis/credentials/serviceaccountkey). Select the default service account or create a new one, give it a name, select Project then Owner as the role, JSON as the key type, and click Create. This downloads a JSON file with all the credentials that will be needed for Terraform to manage the resources. Copy the downloaded JSON file to the *azure_arc_servers_jumpstart/gcp/terraform* directory.
+* Once the new project is created and selected in the dropdown at the top of the page, you must enable Compute Engine API access for the project. Click on "+Enable APIs and Services" and search for "Compute Engine". Then click Enable to enable API access.
 
-* Finally, make sure your SSH keys are available in *~/.ssh* and named *id_rsa.pub* and *id_rsa*. If you followed the ssh-keygen guide above to create your key then this should already be setup correctly. If not, you may need to modify [*main.tf*](../gcp/terraform/main.tf) to use a key with a different path.
+    ![](../img/gcp_windows/03.png)
+
+    ![](../img/gcp_windows/04.png)
+
+* Next, set up a service account key, which Terraform will use to create and manage resources in your GCP project. Go to the [create service account key page](https://console.cloud.google.com/apis/credentials/serviceaccountkey). Select "New Service Account" from the dropdown, give it a name, select Project then Owner as the role, JSON as the key type, and click Create. This downloads a JSON file with all the credentials that will be needed for Terraform to manage the resources. Copy the downloaded JSON file to the [*azure_arc_servers_jumpstart/gcp/ubuntu/terraform*](../gcp/ubuntu/terraform/) directory.
+
+    ![](../img/gcp_windows/05.png)
+
+* Finally, make sure your SSH keys are available in *~/.ssh* and named *id_rsa.pub* and *id_rsa*. If you followed the ssh-keygen guide above to create your key then this should already be setup correctly. If not, you may need to modify [*main.tf*](../gcp/ubuntu/terraform/main.tf) to use a key with a different path.
```

Comment:
See this section in the below README. I would love for us to be aligned here as much as possible. 
https://github.com/likamrat/azure_arc/blob/master/azure_arc_k8s_jumpstart/docs/gke_terraform.md